x-pack/plugins/apm/public/components/app/Settings/agent_keys/index.tsx

```diff
@@ -235,7 +236,7 @@ function AgentKeysContent({
           <p>
             {i18n.translate('xpack.apm.settings.agentKeys.emptyPromptBody', {
               defaultMessage:
-                'Create keys to authorize agent requests to the APM Server.',
+                'Create APM agent keys to authorize APM agent requests to the APM Server.',
```

Comment:
```suggestion
                'Create APM agent keys to authorize requests to the APM Server.',
```
@bmorelli25 Thoughts on reducing the APM agents mention here? Just reads a little harder